.github/workflows/deploy.yml

```diff
@@ -55,7 +55,7 @@ jobs:
     - name: Install tox
       run: |
         python -m pip install --upgrade pip
-        pip install --upgrade tox
+        pip install tox
```

Comment:
Personally, I'd skip pip and have the entire env with tox pinned via `--constraint=`.

Reply:
I'm not familiar with this, how to skip pip but still get tox installed?

Reply:
I meant skip the previous line and have constraint on this one.